src/components/app/artistscontainer/artistinfocontainer/artist-info-container.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+import ArtistInfo from './artistinfo/artist-info'
+import ArtistAlbums from './artistalbums/artist-albums'
+import RelatedArtists from './relatedartists/related-artists'
+
+@inject('artistInfoStore')
+@observer
+class ArtistInfoContainer extends Component {
+  componentDidUpdate() {
+    document.getElementsByClassName('ai__container')[0].scrollIntoView({
```

Comment:
Nie reactowe :/ Lepiej to zrobić przy użyciu ref'ów imho.

Reply:
oki, done